src/core/src/opsets/opset.cpp

```diff
@@ -192,6 +192,17 @@ const ov::OpSet& ov::get_opset10() {
     return opset;
 }
 
+const ov::OpSet& ov::get_opset11() {
```

Comment:
Please also update `get_available_opsets()` method

Reply:
and all other places as well where last upset is supposed